client/src/featureform/resources_test.py

```diff
@@ -40,6 +40,30 @@ def redis_config():
         db=3,
     )
 
+@pytest.fixture
+def online_blob_config(
```

Comment:
with fixtures, you can pass in another fixture to it. Basically, define the `azure_blob_store_config` fixture first then for this one, just pass it in as a parameter. 

```
@pytest.fixture
def azure_blob_store_config():
      return AzureBlobStoreConfig(
         account_name="<account_name>",
         account_key="<account_key>",
         container_name="examplecontainer",
         root_path="example/path",
     )

@pytest.fixture
def online_blob_config(azure_blob_store_config):
     return OnlineBlobConfig(
         store_type="AZURE",
         store_config=azure_blob.serialize(),
     )
```